docs/src/user_docs/assembly/u32_operations.md

```diff
@@ -13,7 +13,8 @@ For instructions where one or more operands can be provided as immediate paramet
 | ------------- | ----------- | ------------- | ------------------------------------------ |
 | u32test       | [a, ...]    | [b, a, ...]   | $b \leftarrow \begin{cases} 1, & \text{if}\ a < 2^{32} \\ 0, & \text{otherwise}\ \end{cases}$ |
 | u32testw      | [A, ...]    | [b, A, ...]   | $b \leftarrow \begin{cases} 1, & \text{if}\ \forall\ i \in \{0, 1, 2, 3\}\ a_i < 2^{32} \\ 0, & \text{otherwise}\ \end{cases}$ |
-| u32assert     | [a, ...]    | [a, ...]      | Fails if $a \ge 2^{32}$ |
+| u32assert or u32assert.1    | [a, ...]    | [a, ...]      | Fails if $a \ge 2^{32}$, .1 can be omitted |
+| u32assert.2   | [a, b,...]  | [a, b,...]    | Fails if $a \ge 2^{32}$ or $b \ge 2^{32}$ |
```

Comment:
nit: would prover these to be as follows:
```
| u32assert <br> u32assert.1 | [a, ...]    | [a, ...]  | Fails if $a \ge 2^{32}$, .1 can be omitted |
| u32assert.2   | [b, a,...]  | [b, a,...] | Fails if $a \ge 2^{32}$ or $b \ge 2^{32}$ |
```
This would be more consistent with how we do it in other places.

Reply:
Sure. I'll update the mdbook to make it consistent.